packages/react-core/src/components/Menu/MenuItem.tsx

```diff
@@ -24,6 +24,10 @@ export interface MenuItemProps extends Omit<React.HTMLProps<HTMLLIElement>, 'onC
   itemId?: any;
   /** Target navigation link. Should not be used if the flyout prop is defined. */
   to?: string;
+  /** Navigation link target. Only set when the to property is present. */
```

Comment:
```suggestion
  /** Navigation link target. Only set when the to property is present. If isExternalLink is also passed in, this property will be set to "_blank" */
```

Or something along those lines, just to let consumers aware that this'll be automatically set for them when passing isExternalLink in